package.json

```diff
@@ -29,8 +29,10 @@
     "compile": "microbundle build -f modern,umd --globals react=React",
     "postcompile": "cp dist/index.modern.mjs dist/index.modern.js && cp dist/index.modern.mjs.map dist/index.modern.js.map",
     "test": "run-s eslint tsc-test jest",
+    "test:debug": "run-s eslint tsc-test jest:debug",
     "eslint": "eslint --ext .js,.ts,.tsx .",
     "jest": "jest",
+    "jest:debug" : "node --inspect ./node_modules/jest/bin/jest.js --runInBand",
```

Comment:
How would you use these new scripts?
What kind of errors are you debugging?

Reply:
i just run `pnpm test:debug` then open chrome's node debugger to step through the tests, inspect values, etc. It was helpful when i couldn't quite figure out why my test was failing

Reply:
Smart, seems more efficient than trying to `console.log` debug these kinds of errors :+1: